addons/website_event_crm/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Website Events CRM',
+    'version': '1.0',
+    'category': 'Website/Website',
+    'website': 'https://www.odoo.com/page/events',
+    'description': "Display different types of lead creation for the rule creation",
```

Comment:
It's more 'active the lead creation basis 'per order'' right ?
Maybe we could add it in the description to be a bit more specific